sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -155,6 +161,7 @@ def _add_argparse_args(cls, parser):
 
 def model_pcollection(argv):
   """Creating a PCollection from data in local memory."""
+  # [START model_pcollection]
```

Comment:
I do not think this needs to go up and include the definition for options to show how `Create` works.